test/sharness/t0060-daemon.sh

```diff
@@ -8,33 +8,31 @@ test_description="Test daemon command"
 
 . lib/test-lib.sh
 
-gwyport=8080
-apiport=5001
+gwyport=8082
+apiport=5002
+swarmport=4003
+
+API_MADDR="/ip4/0.0.0.0/tcp/$apiport"
```

Comment:
I need them to be set here first, so I can provide them to `ipfs init/config` **before** the daemon gets run.